android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1302,6 +1302,11 @@ else if (e.type == ExoPlaybackException.TYPE_SOURCE) {
         if (!isBehindLiveWindow(e)) {
             updateResumePosition();
         }
+
+        Throwable errorCause = e.getSourceException();
```

Comment:
no need to define variable for this just use e.getSourceException() directly in the if statement, right ?